src/main/webapp/i18n/de/imprint.json

```
@@ -0,0 +1,33 @@
+{
+  "imprint": {
+    "headline": "Impressum",
+    "publisher": {
+      "headline": "Herausgeber",
+      "contact": "Technische Universität München<br>Postanschrift: Arcisstraße 21, 80333 München<br>Telephone: <a href='tel:+498928901' class='external-link'>+49-(0)89-289-01</a><br>Telefax: +49-(0)89-289-22000<br>Email: <a href='mailto:poststelle@tum.de' class='external-link'>poststelle(at)tum.de</a>"
```

please use @ and not (at)

Please change this for the english translation as well
```suggestion
      "contact": "Technische Universität München<br>Postanschrift: Arcisstraße 21, 80333 München<br>Telephone: <a href='tel:+498928901' class='external-link'>+49-(0)89-289-01</a><br>Telefax: +49-(0)89-289-22000<br>Email: <a href='mailto:poststelle@tum.de' class='external-link'>poststelle@tum.de</a>"
```